### PR TITLE
[AIFlow] Add workflow execution on event demo

### PR DESCRIPTION
**Pull request by Sxnan**
## What is the purpose of the change

Add workflow execution on event demo

## Brief change log

- Add workflow execution on event demo

## Verifying this change

demo is manually tested.

### CODE REVIEW

File: flink-ai-flow/examples/workflow_on_event/workflows/daily_workflow/daily_workflow.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        return [[os.path.join(daily_dir, file_name) for file_name in file_names]]
+
+
+class DataProcessingProcessor(PythonProcessor):
+
+    def process(self, execution_context: ExecutionContext, input_list: List):
+        file_paths = input_list[0]
+        res = 0
+        for file_path in file_paths:
+            with open(file_path, 'rt') as f:
```
Comment (bgeng777) [inline flink-ai-flow/examples/workflow_on_event/workflows/daily_workflow/daily_workflow.py:55]: 't' is a redundant config for open() method as text model is the default choice.

**Review by bgeng777 — APPROVED** [review]
As an example, it would be better if we can add READEME and more detailed comments.

File: flink-ai-flow/examples/workflow_on_event/workflows/daily_workflow/daily_workflow.py (hunk 1 above)
Comment (Sxnan) [reply flink-ai-flow/examples/workflow_on_event/workflows/daily_workflow/daily_workflow.py]: fixed in 7241764

File: flink-ai-flow/examples/workflow_on_event/workflows/hourly_workflow/hourly_workflow.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        add_event(namespace=project_name, event_type='DATA_EVENT',
+                  event_key='hourly_data', event_value='ready', sender=ANY_CONDITION)
+    af.workflow_operation.start_new_workflow_execution_on_events(workflow_name, event_conditions=[condition])
+
+
+if __name__ == '__main__':
+    # main()
+    af.init_ai_flow_context()
+    project_name = af.current_project_config().get_project_name()
+    af.get_ai_flow_client().send_event(BaseEvent(key='hourly_data',
```
[the remark this answers is not in the document]
Comment (Sxnan) [reply flink-ai-flow/examples/workflow_on_event/workflows/hourly_workflow/hourly_workflow.py]: fixed in 7241764

**Pull request merged**